### ISSUE TITLE
Question about BN in nas_fpn

### ISSUE THREAD

**Issue by ggjy** [DetectionTeamUCAS/NAS_FPN_Tensorflow#12]
Hi, thanks for your code.
I try to implement NAS-FPN in pytorch, but there seems to be some errors with my BN layer (Both 1-stage(RetinaNeet) / 2-stage). I freeze the BN layer in r50 backbone, fine-tune the BN in NAS-FPN (initialized with weight=1 bias=0), am I right?
The log is like this, training seems OK, but eval is always 0 (0.001).

Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.000
2019-08-08 01:31:15,314 - INFO - Epoch [1][7330/7330]	lr: 0.02000, bbox_mAP: 0.0000, bbox_mAP_50: 0.0000, bbox_mAP_75: 0.0000, bbox_mAP_s: 0.0000, bbox_mAP_m: 0.0000, bbox_mAP_l: 0.0000, bbox_mAP_copypaste: 0.000 0.000 0.000 0.000 0.000 0.000
2019-08-08 01:31:39,196 - INFO - Epoch [2][50/7330]	lr: 0.02000, eta: 8:01:08, time: 0.477, data_time: 0.070, memory: 4057, loss_rpn_cls: 0.0640, loss_rpn_bbox: 0.0223, loss_cls: 0.1917, acc: 96.7520, loss_bbox: 0.0561, loss: 0.3341
2019-08-08 01:31:56,286 - INFO - Epoch [2][100/7330]	lr: 0.02000, eta: 8:00:42, time: 0.342, data_time: 0.007, memory: 4057, loss_rpn_cls: 0.0718, loss_rpn_bbox: 0.0185, loss_cls: 0.2137, acc: 96.5977, loss_bbox: 0.0619, loss: 0.3659

**Comment by ggjy** [comment]
There is another little thing. The RetinaNet (P2 to P6 in your code) is different with the paper (P3 to P7), will (P2 to P6) better than (P3 to P7)?
Thanks again~
